dev/tools/gen_defaults/lib/template.dart

```diff
@@ -43,6 +43,12 @@ abstract class TokenTemplate {
     return _tokens[tokenName];
   }
 
+  /// Resolve a token if available or null otherwise. Unlike [getToken], this
+  /// method will not log the token lookup if the token doesn't exist.
+  dynamic getTokenOrNull(String tokenName) {
```

Comment:
I think it would make more sense to combine this with `getToken`, and have it determine whether to log or not. This would also simplify the code for `color` below

Reply:
Do you mean with a named parameter?

Something like `dynamic getToken({bool optional=false})` ?

Reply:
I implemented that aproach in [cdf469b](https://github.com/flutter/flutter/pull/140872/commits/cdf469b99201472b50213af43c878cbbe2db13ee)